test_backend.py

```diff
@@ -191,10 +185,17 @@ def test_robot_move(input_coordinates, input_direction, distance, output_coordin
 def test_tile_size(map_name):
     """
     Take size of tiles used in JSON files and assert correct tile size.
-
     This test has to be removed, when width and height of tile image are
     no longer constants used for tile drawing.
     """
     data = get_data("maps/" + map_name + ".json")
     assert data["tilewidth"] == 64
     assert data["tileheight"] == 64
+
+@pytest.mark.parametrize("map_name", ["test_3", "test_5"])
+def test_map_is_valid(map_name):
+    assert check_squares(map_name) == True
+
+@pytest.mark.parametrize("map_name", ["test_6"])
+def test_map_is_invalid(map_name):
+    assert check_squares(map_name) != True 
```

Comment:
pro tip: ta funkce vrací `bool`, takže stačí `assert check_squares(map_name)` a `assert not check_squares(map_name)`